docs/userGuide/syntax/siteNavigationMenus.mbdf

```diff
@@ -3,26 +3,23 @@
 **A _Site Navigation Menu_ (==_siteNav_ for short==) is a fixed menu on the ==left edge== of a page**, that can be used to show a road map of the main pages of your site.
 
 Steps to add a siteNav:
-1. Format your siteNav as an unordered Markdown list and save it inside the `_markbind/navigation` directory.
+1. Format your siteNav as an unordered Markdown list and save it inside the `_markbind/navigation` directory. You may include dividers in the siteNav via <a target="_blank" href="{{ baseUrl }}/userGuide/readerfacingfeatures#horizontal-rules">horizontal rules</a>.
 2. Specify the file as the value of the `siteNav` attribute in the `<frontmatter>` of the page.
 
 <div class="indented">
 
 {{ icon_example }} Here is an example siteNav code saved in **`_markbind/navigation/`**`mySiteNav.md` file:
 
-<box>
-
-<code>
-* [:house: Home]({{ showBaseUrlText }}/index.html)<br>
-* Docs<br>
-&nbsp;&nbsp;* [User Guide]({{ showBaseUrlText }}/ug.html)<br>
-&nbsp;&nbsp;* [Dev Guide]({{ showBaseUrlText }}/dg.html)<br>
-* [Search]({{ showBaseUrlText }}/search.html)<br>
-&nbsp;&nbsp;* [Google Search](https://www.google.com/)<br>
-&nbsp;&nbsp;* [YouTube Search](https://www.youtube.com/)<br>
-* [Contact]({{ showBaseUrlText }}/contact.html)
-</code>
-</box>
+```{.no-line-numbers}
```

Comment:
Do you think adding a divider in the example would be helpful, since we mentioned it above?

Reply:
I think we should. So that they can understand how it looks and accordingly decide whether to use or not?

Reply:
What do you think about this?

Reply:
Sorry, I don't see any changes here?
<img width="784" alt="Screenshot 2020-03-04 at 12 19 05 PM" src="https://user-images.githubusercontent.com/17447681/75844886-582d2f80-5e12-11ea-9964-a7c5aa4524b9.png">

Reply:
Ah my bad. I was asking you whether to do it or not. Didn't realise you were saying to implement and show. I have done it now. Sorry for the confusion.

Reply:
Oops, sorry for the lack of clarity!